sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/metadata.yaml

```
@@ -0,0 +1,23 @@
+---
+friendly_name: Data from external source Adjust
+description: >
+  This table represents data downloaded from Adjust. Please note that the API token used is Marlene Hirose's personal Adjust API token
+owners:
+  - mhirose@mozilla.com
+labels:
+  incremental: false
+scheduling:
+  dag_name: bqetl_adjust_derived
```

```suggestion
  dag_name: bqetl_adjust
```

unless there is a good reason for having an separate DAG

---

it was slightly different than what we had previously. One was in marketing-prod, this one is in shared prod. Will use the old DAG